prov/efa/src/rxr/rxr_env.c

```diff
@@ -146,7 +156,7 @@ void rxr_env_define()
 	fi_param_define(&rxr_prov, "enable_shm_transfer", FI_PARAM_INT,
 			"Enable using SHM provider to perform TX operations between processes on the same system. (Default: 1)");
 	fi_param_define(&rxr_prov, "use_device_rdma", FI_PARAM_INT,
-			"whether to use device's RDMA functionality for one-sided and two-sided transfer.");
+			"whether to use device's RDMA functionality for one-sided and two-sided transfer. (Default: %d)", rxr_env.use_device_rdma);
```

Comment:
I think we can check device capability here. If the device does not support RDMDa, print something like "EFA device does not support RDMA, so you cannot set this variable to 1".

Reply:
Done.